test/cli/dev.js

```diff
@@ -73,7 +73,7 @@ test('`fusion dev` works with assets', async t => {
     t.equal(await request(`http://localhost:${port}/filename`), 'src/main.js');
     t.equal(
       await request(`http://localhost:${port}/json`),
-      '/_static/7526e1bdce8d3d115d6b4d6b79096e1c.json'
+      '/_static/20355efabaae9ed4d51fbc5a68eb4ce3.json'
```

Comment:
This is also the same hash that is produced via `md5sum`. Not sure what the old hash algorithm was.